bin/send_message.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eu
+file_name=${1:-"example1.json"}
+environment=${2:-"sandbox"}
+file="../src/test/resources/${file_name}"
+echo "using file: ${file_name} environment:${environment}"
+shared_access_signature="SharedAccessSignature sr=sb%3A%2F%2Fbulk-scan-servicebus-sandbox.servicebus.windows.net%2F&sig=Cz5XsYG8tGYUkY0nwrtO%2Fkxc2YOPTsoDyrJ%2BwctPWEU%3D&se=1538398077&skn=SendSharedAccessKey"
```

Comment:
should this be here?

Reply:
no it needs to be calculated / passed in

Reply:
yeah ... its a first draft. we can enhance this later ?

Reply:
I am mostly concerned with the security aspect